backend/src/app.js

```diff
@@ -1,16 +1,25 @@
 const express = require("express");
+
 //create express app
 const app = express();
 //require knex function to communicate with database
 const knex = require("./database/connection");
 
 //configure app
+//handle cors error (temporary fix) 
+ app.use(function(req, res, next) {
```

Comment:
app is now using cors from recent changes

Reply:
can most likely pull in changes from development and remove this section

Reply:
got it, I went ahead and removed my changes to app.js to resolve the conflicts.